app/views/account/AccountSettingsView.coffee

```diff
@@ -83,6 +84,12 @@ module.exports = class AccountSettingsView extends CocoView
       confirmModal.on 'confirm', @resetProgress
       @openModalView confirmModal
 
+  onClickResendVerificationEmail: (e) ->
+    $.post "/db/user/#{me.id}/request-verify-email", ->
+      link = $(e.currentTarget)
+      link.find('.resend-text').addClass('hide')
+      link.find('.sent-text').removeClass('hide')
```

Comment:
We shouldn't be putting URLs in views anymore. This should go in a `User` method which constructs the URL.